src/Jackett.Common/Indexers/EraiRaws.cs

```diff
@@ -16,25 +17,22 @@ namespace Jackett.Common.Indexers
 {
     public class EraiRaws : BaseWebIndexer
     {
-        const string RSS_PATH = "rss-all-magnet";
-
-        private readonly IReadOnlyDictionary<string, int> sizeEstimates = new Dictionary<string, int>() {
-            { "1080p", 1332 }, // ~1.3GiB
-            { "720p", 700 },
-            { "540p", 350 }
-        };
+        const string RSS_PATH = "feed/?type=magnet";
 
         public override string[] AlternativeSiteLinks { get; protected set; } = {
-            "https://www.erai-raws.info/",
-            "https://erairaws.nocensor.space/"
+            // At some point the beta site will probably replace the current one
+            // At that point, these can probably be re-enabled.
+            // "https://www.erai-raws.info/",
+            // "https://erairaws.nocensor.space/"
         };
 
         public EraiRaws(IIndexerConfigurationService configService, Utils.Clients.WebClient wc, Logger l,
             IProtectionService ps, ICacheService cs)
             : base(id: "erai-raws",
                    name: "Erai-Raws",
                    description: "Erai-Raws is a team release site for Anime subtitles.",
-                   link: "https://www.erai-raws.info/",
+                   //link: "https://www.erai-raws.info/",
```

Comment:
you can use the LegacySiteLinks override to automatically update users configs from old domain to new default domain
see for example
https://github.com/Jackett/Jackett/blob/8b59fb0a5281f280a50e34c7165897e43c36980f/src/Jackett.Common/Indexers/MoreThanTV.cs#L29-L31

Reply:
Nice! Updated